src/HotChocolate/ElasticSearch/src/Data.ElasticSearch.Driver/Paging/ElasticSearchExecutablePagingContainer.cs

```diff
@@ -0,0 +1,52 @@
+﻿using HotChocolate.Data.ElasticSearch.Execution;
+using HotChocolate.Types.Pagination;
+
+namespace HotChocolate.Data.ElasticSearch.Paging;
+
+public class ElasticSearchExecutablePagingContainer<TEntity> : IElasticSearchPagingContainer<TEntity>
```

Comment:
I dont think the IElasticSearchPaginationContainer abstraction is necessary here. I belive we have this in MongoDB because find fluend and aggregate fluent behaves differently. I believe we could just use the IElasticSearchExecutable here. Just add WithSkip and WithTake to the executable and we should be all good (and can remove the pagination container abstraction)

Reply:
I will move the code

Reply:
I moved it